kubernetes/linux/setup.sh

```diff
@@ -59,7 +59,7 @@ sudo tdnf install jq-1.6-1.cm2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo tdnf install libcap -y
 
-sudo tdnf install telegraf-1.28.5 -y
+sudo tdnf install telegraf-1.29.4 -y
```

Comment:
Please validate the perf and functional tests for this update, and also review any issues on this telegraf version related to promethus, disk, diskio plugins.

Reply:
lot of work to individually test all this. I was hoping to use CI/CD clusters for this. WDYT?

Reply:
> lot of work to individually test all this. I was hoping to use CI/CD clusters for this. WDYT?

Can you please run some basic tests on both Linux and Windows to ensure this doesnt have any breaking change. Primarily for windows, we have seen issues  when we upgraded telegraf.